src/main/java/de/infynyty/zuap/insertion/WGZimmerInsertion.java

```diff
@@ -40,24 +38,24 @@ public WGZimmerInsertion(final @NotNull Element element) throws IllegalStateExce
     @Override
     protected SortedMap<String, Optional<String>> setProperties() {
         SortedMap<String, Optional<String>> map = new TreeMap<>();
+        map.put("Region", Optional.of(setRegion()));
         map.put("Rent", Optional.of(String.valueOf(setRent())));
         map.put("Move-in Date", Optional.of(new SimpleDateFormat("dd.MM.yyyy").format(setMoveInDate())));
         map.put("Next Tenant Wanted", Optional.of(setIsNewTenantWanted() ? "Yes" : "No"));
         return map;
     }
 
+    private String setRegion() {
```

Comment:
Currently, this does not seem to return all the correct information. For example [this insertion](https://www.wgzimmer.ch/wglink/de/9cb8f10e-6e66-4501-82d5-32b33861b57f/bern/15-8-2023-700-bern0.html) has region data, address data, city data, county data and data about nearby infrastructure, but within the actual insertion only the region, the city and the nearby infrastructure appear (i.e. all present data: `Region Bern Adresse Bernstrasse 121 Ort 3072 Ostermundigen Kreis / Quartier Ostermundigen In der Nähe  LIDL, BUS 10`, but only `Bern, Ostermundigen LIDL, BUS 10` is actually posted).

Reply:
The data is pulled only from the search results page and it does not have the data you mentioned. This is done to minimize the number of calls to the server, because otherwise captcha gets triggered.

Reply:
I would keep it like this for now, if it becomes an issue, we can fix it.